tests/Database/DatabaseEloquentIntegrationTest.php

```diff
@@ -1296,15 +1296,19 @@ public function testTimestampsUsingOldSqlServerDateFormat()
 
     public function testTimestampsUsingOldSqlServerDateFormatFailInEdgeCases()
```

Comment:
I guess since the method doesn't fail anymore, the name should be adapted maybe?

Reply:
Yep, thanks.